src/main/java/org/dita/dost/module/CopyToModule.java

```diff
@@ -251,7 +251,7 @@ private void copyFileWithPIReplaced(final URI src, final URI target, final URI c
         public void startElement(final String uri, final String localName, final String qName, final Attributes atts)
                 throws SAXException {
             Attributes resAtts = atts;
-            if ((TOPIC_XREF.matches(atts) || TOPIC_LINK.matches(atts) || TOPIC_IMAGE.matches(atts))
+            if ((TOPIC_XREF.matches(atts) || TOPIC_LINK.matches(atts) || TOPIC_IMAGE.matches(atts) || SVG_D_SVGREF.matches(atts))
```

Comment:
I don't think the addition here does anything, because `<svgref>` is specialized from `xref` so should already match the first condition.

Reply:
👍